database-seed/remote-image-getter.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+
+const uri = 'https://loremflickr.com/';
+const productTypes = ['jewellery','dress','hat','cargopants','craftproduct','kitten','necklace','purse','shoes','chair','socks','computer','toys','collectible'];
+const resolutions = [[800,600],[600,600],[1200,800],[800,1200],[300,400],[200,150]];
+
+const generateImages = (num) => {
+    for (let i = 0; i < num; i++) {
+        let prodID = `${i}`;
+        let destination = path.join(__dirname, `/images/${prodID}/`);
+        let filename = `${prodID}.jpg`;
+        let resolution = randResolution();
+        let type = randType();
+        let url = uri+resolution+type;
```

Comment:
what is the purpose of creating this url? I see you use it in your axios request. You send a GET request to the url and when you get the response, that is when the file with the image gets created?